src/lib/kfs_link.c

```diff
@@ -0,0 +1,10 @@
+#include "kfs_link.h"
+
+int kfs_link(const char* first_path, const char* second_path) {
+    char *path1 = local_disk_cache_path(first_path);
+    char *path2 = local_disk_cache_path(second_path);
```

Comment:
it's correct only if first_path and second_path inside our mount foulder, but what if first or second path is in normal file system?